.buildkite/pipeline.yml

```diff
@@ -288,6 +288,7 @@ steps:
                 - main.go
 
                 - .buildkite/integration.pipeline.yml
+                - .buildkite/bk.integration.pipeline.yml
```

Comment:
What's the difference between `integration.pipeline.yml` and `bk.integration.pipeline.yml`? I wonder if, in a follow-up, the name can be changed to a more semantic name.